packages/amplify-category-api/src/index.ts

```diff
@@ -224,9 +224,6 @@ export async function executeAmplifyCommand(context: $TSContext) {
   } catch (error) {
     if (error) {
       printer.error(error.message || error);
-      if (error.stack) {
-        printer.info(error.stack);
```

Comment:
might be helpful to change this to `printer.debug(error.stack);`

Then we can see the stack traces when `--debug` is passed, but they're be hidden by default

Reply:
Fixed